src/i18n/locales/en/translation.json

```diff
@@ -717,10 +722,11 @@
     }
   },
   "show_utxos": {
-    "select_utxos": "Select UTXOs",
-    "considered_utxos": "Considered UTXOs",
-    "show_utxo_title": "Select UTXOs to be considered",
-    "show_utxo_subtitle": "The following UTXOs are considered in the transaction. Every unselected UTXO will be frozen and can be unfrozen later on.",
-    "show_utxo_subtitle_when_allutxos_are_frozen": "The following UTXOs are frozen. Please select them to be considered in the transaction."
+    "text_select_utxos_tooltip": "Select UTXOs",
+    "title": "Select eligible UTXOs",
+    "subtitle_zero": "Please select one or more UTXOs for possible inclusion in the transaction.",
+    "subtitle_one": "The selected UTXO is eligible for inclusion in the transaction.",
```

Comment:
I see the matching with the subtitle_other, but it reads a bit weird imo "is eligible for inclusion" as it will for sure be included,
maybe something like `The selected UTXO will be used in the transaction.`

Reply:
I open to any suggestion. However, doesn't 

> The selected UTXO will be used in the transaction.

Sound more like it will "definitely used" in the transaction than the other phrase ("eligible for inclusion")?

Reply:
Ahh.. okay. Now I think I get what you mean. If only one UTXO is selected, it will of course be included, yes?
Good point! I am not sure if the phrasing should change so "suddenly" when a user ticks some boxes in this component as it is just for quickly freezing/unfreezing..

I think @editwentyone might have good inputs to this also.

Reply:
@MarnixCroes I can't tell right now what is the best way, I would try to avoid the switching during selecting. lets see when we keep it like this first and then we can adjust it in another PR if needed

Reply:
@MarnixCroes Okay for you if it is merged the way it is currently?
I think it the context of "quickly freezing/unfreezing UTXOs", phrasing it this way is "okay".

Reply:
Hey @MarnixCroes - if it is okay for you, I'll merge this soon.